lib/Runtime/PlatformAgnostic/Platform/Linux/DateTime.cpp

```diff
@@ -193,23 +190,16 @@ namespace DateTime
 
         tzset();
         time_t ltime = timegm(&utc_tm);
-        struct tm local_tm;
+
+        struct tm local_tm = {0};
         localtime_r(&ltime, &local_tm);
 
         TM_TO_YMD((&local_tm), local, leap_added, utc->year);
         // put milliseconds back
         local->time += milliseconds;
 
-        // ugly hack but;
-        // right in between dst pass
-        // we need mktime trick to get the correct dst
-        utc_tm.tm_isdst = 1;
-        ltime = mktime(&utc_tm);
-        ltime += utc_tm.tm_gmtoff;
-        localtime_r(&ltime, &utc_tm);
-
-        isDaylightSavings = utc_tm.tm_isdst;
-        offset = utc_tm.tm_gmtoff / 60;
+        isDaylightSavings = local_tm.tm_isdst ? 0 : 1;
```

Comment:
>isDaylightSavings = local_tm.tm_isdst ? 0 : 1; [](start = 8, length = 46)

"isDaylightSavings" is of type "bool", why assign "0/1" instead of "false/true"?